src/Component/AutoMapper/AutoMapper.php

```diff
@@ -186,14 +186,16 @@ public static function create(
         string $classPrefix = 'Mapper_',
         bool $attributeChecking = true,
         bool $autoRegister = true,
-        string $dateTimeFormat = \DateTime::RFC3339
+        string $dateTimeFormat = \DateTime::RFC3339,
+        bool $allowReadOnlyTargetToPopulate = false
```

Comment:
since the automapper is created for you by the bundle, shouldn't this value be a configuration in the bundle?